Tests/Config/Settings.ps1

```diff
@@ -2,7 +2,7 @@
 # Licensed under the MIT License.
 
 # The account that the tests will be running under
-$script:ownerName = 'PowerShellForGitHubTeam'
+$script:ownerName = 'jorivtest'
```

Comment:
Are these intentionally changed?

Reply:
I think you are looking at an older revision.